MITADS/utils/line_rules.py

```diff
@@ -4,7 +4,11 @@
 
 
 class LineRules:
-    """Rules to identify a valid line"""
+    """Rules to identify a valid line
+    Lines Rules has the ability to automatically log the discarded sentences in a json file.
+    To use it you need to pass a file path to the __init__ and then call save when you have finished to write to disk.
+    The file is a dict where keys are the rule and values are a list of discarded sentences"""
+
 
     def __init__(self, discard_file=None):
```

Comment:
I am wondering if it is better a parameter to the script or just a default file name (maybe a txt file with the same script name except the extension in the output folder)?

In this way for the debug doesn't require any code change that maybe someone will commit as error.